lib/logger/test/logger/utils_test.exs

```diff
@@ -19,6 +19,9 @@ defmodule Logger.UtilsTest do
       assert compute_mode(:async, 20, 15, 20, 7500, 10000) == :sync
       assert compute_mode(:async, 30, 15, 20, 7500, 10000) == :sync
       assert compute_mode(:async, 8000, 15, 20, 7500, 10000) == :sync
+      assert compute_mode(:async, 0, 15, 0, 7500, 10000) == :sync
```

Comment:
It should be:

    compute_mode(:async, 0, 0, 0, 7500, 10000)

That's because the number 15 is always computed in function of `sync_threshold`. We also need to change the examples below. Thanks!

Reply:
okay, also the build is failing in OTP 19.1, is that something which can be avoided?